composer.json

```diff
@@ -22,8 +22,7 @@
         "symfony/console": "~2.8",
         "neos/composer-plugin": "^2.0.0",
         "typo3fluid/fluid": "~2.1.3",
-        "ext-mbstring": "*",
-        "ext-xml": "*"
```

Comment:
is the ext-xml removed intentionally? That was added with #1357

Reply:
Ok, then it must be added to `Neos.Flow/composer.json` as well!